.github/workflows/build_and_test.yml

```diff
@@ -25,7 +25,7 @@ on:
       java:
         required: false
         type: string
-        default: 8
+        default: 8.0.345
```

Comment:
@HyukjinKwon @Yikun Can the version be written like this

Reply:
so the actual version was changed from time to time?

Reply:
Yes, always use the latest version

Reply:
got it, thanks

Reply:
https://github.com/actions/setup-java/blob/main/src/distributions/zulu/installer.ts#L30-L45

Yes, we are using zulu, and it find the latest java.

As we offline discussed, we could first to pin java8/java11/java17 to specific version to recover CI.